app/models/energy/ScheduleRuleset.py

```diff
@@ -0,0 +1,113 @@
+"""Schedule Ruleset Schema"""
+from pydantic import BaseModel, Schema, validator, ValidationError, UrlStr, ConstrainedStr
+from typing import List, Union
+from enum import Enum
+from uuid import UUID, uuid4
+from datetime import datetime
+from app.models.energy.ScheduleBase import ScheduleContinuous, ScheduleDiscrete, ScheduleNumericType, ScheduleUnitType, DateTime, Date, Time
+
+
+class ScheduleTypeLimits (BaseModel):
+    """Specifies the data types and limits for values contained in schedules."""
+
+    type: Enum('ScheduleTypeLimits', {'type': 'ScheduleTypeLimits'})
+
+    name: str = Schema(
+        ...,
+        regex=r'^[\s.A-Za-z0-9_-]*$',
+    )
+
+    lower_limit_value: float = Schema(
+        0,
+        description='Lower limit for the schedule type is entered.'
+    )
+
+    upper_limit_value: float = Schema(
+        1,
+        description='Upper limit for the schedule type is entered.'
+    )
+
+    numeric_type: ScheduleNumericType
+
+    unit_type: ScheduleUnitType
+
+
+class DayValue(BaseModel):
```

Comment:
@macumber , I see that the standards gem just stores the day values as a list:
![image](https://user-images.githubusercontent.com/5567574/59862659-2ee67500-9351-11e9-9b07-e27f8df692f9.png)
Do you usually do the conversion of this list into a set of "values until time" under the hood of OpenStudio?  If so, would you also recommend that we do this same conversion on the Honeybee end?

Reply:
If you will always use hourly values then sure, its probably easiest to just store a list.

Reply:
There will likely be cases where we want sub-hourly values so let's forego this for now and I will write translators from the standards gem values to the schema values.